build.sbt

```diff
@@ -1,8 +1,16 @@
+enablePlugins(GitVersioning)
+
 organization := "org.scalaz.stream"
 
 name := "scalaz-stream"
 
-version := "snapshot-0.7"
+val ReleaseTag = """^release/([\d\.]+a?)$""".r
+git.gitTagToVersionNumber := {
+  case ReleaseTag(version) => Some(version)
+  case _ => None
+}
+
+git.formattedShaVersion := git.gitHeadCommit.value      // bintray really hates -SNAPSHOT versions, so we just use the hash
```

Comment:
> I think sbt-git does this only if there are uncommitted changes.

Nope, it does it all the time.  Comment out the line to see!  :-)

> If it is still necessary could you change the comment to something more informative like "bintray does not allow -SNAPSHOT versions ..."?

I could change the comment if you would like.  Strictly speaking, bintray *does* allow SNAPSHOT versions, you just need to configure your repo a little differently.  Given that we're publishing *stable* snapshots though (i.e. we're not republishing), the `-SNAPSHOT` prefix would actually do more harm than good by defeating Maven's (and Ivy's) caching assumptions.